test/clj/rems/api/testing.clj

```diff
@@ -53,7 +53,7 @@
 
 (defn coll-is-not-empty? [data]
   (and (coll? data)
-       (not (empty? data))))
+       (seq data)))
```

Comment:
note that now this function returns `false`, `nil`, or a sequence. I'd prefer it to return a boolean.

Reply:
I refactored this so that the duplication is removed and therefore also this problem.